engine/schema/src/main/resources/META-INF/db/views/cloud.volume_view.sql

```diff
@@ -39,6 +39,8 @@ SELECT
     `volumes`.`path` AS `path`,
     `volumes`.`chain_info` AS `chain_info`,
     `volumes`.`external_uuid` AS `external_uuid`,
+    `volumes`.`passphrase_id` AS `passphrase_id`,
```

Comment:
```suggestion
```

Reply:
> @JoaoJandre , are your concerns met?

@DaanHoogland no, we should apply @weizhouapache's suggestion here, no need to have this view changed for now.

Reply:
sorry @JoaoJandre , I think we crossed messages (with Kiran in the mix) here. I don't see `passphrase_id` in the PR. so that means it is applied, right?

If not we'll need to create a new PR.

Reply:
Yes, @DaanHoogland, it was applied in the last commit, before merging.